ecmd-core/pyecmd/__init__.py

```diff
@@ -0,0 +1,236 @@
+# IBM Confidential
```

Comment:
@fenkes-ibm could you update these prologs to be similar to what is in ecmdClientCapi.H instead of the IBM Confidential text.